projects/plugins/crm/src/automation/class-automation-workflow.php

```diff
@@ -218,45 +242,49 @@ private function get_step_class( array $step_data ): string {
 	}
 
 	/**
-	 * Turn on the workflow
+	 * Turn on the workflow.
 	 */
```

Comment:
> Every function, hook, class, and method should have a corresponding @\since version associated with it...
> [source](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#since-section-changelogs)

So we should add `@since $$next-version$$` to all methods in all classes (and interfaces) as well